Makefile

```diff
@@ -1,23 +1,23 @@
-validate-dydx-testnet-3-gentx:
-	docker build --platform linux/amd64 --tag public-testnet-validate $(CURDIR)
+validate-dydx-mainnet-1-gentx:
+	docker build --platform linux/amd64 --tag public-mainnet-validate $(CURDIR)
 	docker run --platform linux/amd64 \
-		-e VALIDATE_GENESIS_DIR_PATH=./dydx-testnet-3 \
+		-e VALIDATE_GENESIS_DIR_PATH=./dydx-mainnet-1 \
 		-e VALIDATE_GENESIS_TAR_URL='https://github.com/dydxprotocol/v4-chain/releases/download/protocol%2Fv0.2.1/dydxprotocold-v0.2.1-linux-amd64.tar.gz' \
 		-e VALIDATE_GENESIS_STAKE_TOKEN=dv4tnt \
 		-e GENESIS_FILE_NAME=pregenesis.json \
 		-e ADD_GENTXS=true \
 		-v $(CURDIR):/workspace \
-		public-testnet-validate 
+		public-mainnet-validate 
 
-validate-dydx-testnet-3-final-genesis:
-	docker build --platform linux/amd64 --tag public-testnet-validate $(CURDIR)
+validate-dydx-mainnet-1-final-genesis:
+	docker build --platform linux/amd64 --tag public-mainnet-validate $(CURDIR)
 	docker run --platform linux/amd64 \
-		-e VALIDATE_GENESIS_DIR_PATH=./dydx-testnet-3 \
+		-e VALIDATE_GENESIS_DIR_PATH=./dydx-mainnet-1 \
 		-e VALIDATE_GENESIS_TAR_URL='https://github.com/dydxprotocol/v4-chain/releases/download/protocol%2Fv0.2.1/dydxprotocold-v0.2.1-linux-amd64.tar.gz' \
 		-e VALIDATE_GENESIS_STAKE_TOKEN=dv4tnt \
 		-e GENESIS_FILE_NAME=genesis.json \
```

Comment:
There are 2 genesis files mentioned: `pregenesis.json` and `genesis.json`. The former exists within the `dydx-mainnet-1` directory, the latter doesn't. Is that OK?

Reply:
yes, I believe `genesis.json` should be generated by the automation scripts, after merging the first gentx PR.